impactlab_tools/os/config.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import yaml
+
+def getConfigDictFromFile(filePath):
+    with open(filePath, 'r') as f:
+        confDict = yaml.load(f)
+    return confDict
```

Comment:
It seems to me that this module will raise an error on load for anyone without a valid yaml file at ../server.yml. That's not OK behavior for a package we're distributing to all users.